docs/developers/references/traits/autoscale.md

```diff
@@ -0,0 +1,100 @@
+# Automatically scale workload by cron or resource utilization
+
+## Prerequisites
+ - [ ] [KEDA v2.0 Beta](https://keda.sh/blog/keda-2.0-beta/)
+ 
+   KEDA will be automatically deployed during veal installation, so just run the following command.
+   ```shell
+   $ vela install
+   ```
+   
+   You can also double check whether KEDA is successfully installed.
+   ```
+   $ helm list -n keda
+   NAME	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART             	APP VERSION
+   keda	keda     	1       	2020-11-02 17:01:16.122247 +0800 CST	deployed	keda-2.0.0-beta1.2	2.0.0-beta
+   ```
+   
+## Scale an application
+   
+- Deploy an application
+
+  Run the following command to deploy application `helloworld`.
+  
+  ```
+  $ vela svc deploy frontend -t webservice -a helloworld --image nginx:1.9.2 --port 80
+  App helloworld deployed
+  ```
+  
+  Check the replicas of Deployment `frontend` which is deployed by workload webservice `helloworld` and there is one replica.
+  
+  (TODO: The command below needs to be replaced  with `vela show` to check the replicas.)
+  ```
+  $ kubectl get deploy frontend
+  NAME       READY   UP-TO-DATE   AVAILABLE   AGE
+  frontend   1/1     1            1           2m52s
```

Comment:
yes，it's a MUST have, before kubecon release. We should print the replica using `vela status` not `vela show`